docs/design/features/globalization-invariant-mode.md

```diff
@@ -104,7 +104,7 @@ Applications can enable the invariant mode by either of the following:
 
     ```xml
     <ItemGroup>
```

Comment:
This needs to be PropetyGroup, not ItemGroup

Reply:
Done.